sdk/src/main/java/com/horizen/account/transaction/EthereumTransaction.java

```diff
@@ -242,7 +242,10 @@ public AddressProposition getTo() {
         throw new RuntimeException(String.format("Invalid to address length %d", to.length));
     }
 
+    @JsonIgnore
     public String getToAddress() {
+        // TODO currently we have a bug that removes 0x prefix during the serialization
```

Comment:
I assume that the `Numeric.toHexString(encodedMessage)` causes this issue, but I'll check while working on RPC.